examples/io/plot_elekta_epochs.py

```diff
@@ -15,35 +15,34 @@
 import mne
 import os
 from mne.datasets import multimodal
-from mne import AcqParserFIF
 
 fname_raw = os.path.join(multimodal.data_path(), 'multimodal_raw.fif')
 
 
 print(__doc__)
 
 ###############################################################################
-# Read raw file and create parser instance
+# Read raw file
 raw = mne.io.read_raw_fif(fname_raw)
-ap = AcqParserFIF(raw.info)
 
 ###############################################################################
 # Check DACQ defined averaging categories and other info
-print(ap)
+print(raw.acqparser)
```

Comment:
I don't like this as everytime you do: raw.acqparser
you call a property and redo the same computation (create instance, parse info etc.)